modules/microvm/virtualization/microvm/netvm.nix

```diff
@@ -14,7 +14,9 @@
       (import ./common/vm-networking.nix {inherit vmName macAddress;})
       ({lib, ...}: {
         ghaf = {
-          users.accounts.enable = lib.mkDefault configHost.ghaf.users.accounts.enable;
+          users.operator.account.enable = lib.mkDefault configHost.ghaf.users.operator.account.enable;
```

Comment:
Does operator need to be enabled in netvm?

Reply:
Currently yes, as `netvm` have `network` user which is system user and which don't allow ssh login. In case user needs to access `netvm` through ssh that is possible using `operator` user.

Reply:
My 2 cents of scoping the user/ssh hardening here. Moving forward, `operator` should **not** be enabled in the netvm. `operator` must use netvm in a way that won't require ssh login nor ssh remote command but limited use-case based RPC instead. We could take at least two strategies to get there:

**a**. Disable `operator` user login and access to `netvm`. Allow remote commands over ssh for `network` system user (instead of `operator`) but do not allow `network` user login. It could be approached via sshd hardening - see for [example](https://www.digitalocean.com/community/tutorials/how-to-harden-openssh-on-ubuntu-20-04#step-3-restricting-the-shell-of-a-user).
**b**. Keep the current way as an intermediate solution and design/implement RPC between `netvm` and `guivm` to replace the ssh tunneled shell command integrations and remote network-manager gui to configure WIFI etc. It's more design and work but would focus **more** on hardening and getting rid of SSH integrations.

Reply:
Great comment - this does little for security (yet). The question is indeed how long we keep the SSH approach before moving on. 

One of the ideas of the global "system users" (not in the sense of `isSystemUser = true;` but as in Ghaf-wide) is to have dedicated user(s) for tasks across machines, separated from the Ghaf login user (`operator`). One reason is to prevent this user to login to the VMs (and host, of course). That implies restricting access to the keys, and only allowing the operator access to a trigger (e.g., via sudo/doas), and add additional hardening as you suggested.